src/main/kotlin/no/nav/familie/ef/sak/repository/domain/Søknad.kt

```diff
@@ -0,0 +1,83 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.kontrakter.ef.søknad.SøknadBarnetilsyn
+import no.nav.familie.kontrakter.ef.søknad.SøknadOvergangsstønad
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Embedded
+import org.springframework.data.relational.core.mapping.MappedCollection
+import org.springframework.data.relational.core.mapping.Table
+import java.util.*
+
+@Table("gr_soknad")
```

Comment:
gr_soknad?

Reply:
BA har delt inn tabeller med prefix `gr` for grunnlag og `po` for noe annet. Hva syns du?

Reply:
Grunnlag er jo ikke såååå langt å skrive? Gr er litt kryptisk.